resource/version.h.in

```diff
@@ -0,0 +1,38 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCUTILS__VERSION_H_
+#define RCUTILS__VERSION_H_
+
+/// \def RCUTILS_VERSION_TIMESTAMP
+/// Defines version file generation time
+#define RCUTILS_VERSION_TIMESTAMP "@RCUTILS_VERSION_TIMESTAMP@"
```

Comment:
I don't think we should have a macro for the timestamp. That implies that the generated code changes on every build which is something generally desired to not happen (see [reproducible builds](https://en.wikipedia.org/wiki/Reproducible_builds)).

Reply:
fixed (timestamp has been removed)